### PR TITLE
feat: Export to share

### PR DESCRIPTION
**Pull request by Vafilor**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/onepanelio/urls.py [hunk 1]
```diff
@@ -16,4 +24,6 @@
     path('get_available_dump_formats', views.get_available_dump_formats),
     path('get_output_path/<int:pk>', views.generate_output_path),
     path('get_annotation_path/<int:pk>', views.generate_dataset_path),
+
+    path('api/opv1/', include(router.urls))
```
Comment (rushtehrani) [inline cvat/apps/onepanelio/urls.py]: does this match our previous naming conventions for routes?

File: cvat-core/src/server-proxy.js [hunk 2]
```diff
@@ -306,7 +306,8 @@
 
             async function exportDataset(id, format) {
                 const { backendAPI } = config;
-                let url = `${backendAPI}/tasks/${id}/dataset?format=${format}`;
+                const opBackendAPI = backendAPI.replace('api/v1', 'onepanelio/api/opv1');
```
Comment (rushtehrani) [inline cvat-core/src/server-proxy.js]: why not `onepanelio/api/v1` since the route is already under `onepanel`?

Comment (Vafilor) [reply cvat-core/src/server-proxy.js]: Will change.

File: cvat/apps/onepanelio/urls.py (hunk 1 above)
Comment (Vafilor) [reply cvat/apps/onepanelio/urls.py]: I was trying to mimic CVAT's engine urls, but I'll do the change pointed out in the comment below.

**Pull request merged**